docs/whats-new/dotnet-docs-mod1.md

```diff
@@ -1,113 +1,115 @@
-ms.custom: October-2022
-ms.date: 11/01/2022
+title: ".NET docs: What's new for January 2023"
+description: "What's new in the .NET docs for January 2023."
+ms.custom: January-2023
+ms.date: 02/01/2023
 ---
 
-# .NET docs: What's new for October 2022
+# .NET docs: What's new for January 2023
 
-Welcome to what's new in the .NET docs for October 2022. This article lists some of the major changes to docs during this period.
+Welcome to what's new in the .NET docs for January 2023. This article lists some of the major changes to docs during this period.
 
 ## .NET breaking changes
 
 ### New articles
 
-- [Authentication in WebAssembly apps](../core/compatibility/aspnet-core/7.0/wasm-app-authentication.md)
-- [Tracking linked cache entries](../core/compatibility/core-libraries/7.0/memorycache-tracking.md)
-- [Fallback file endpoints](../core/compatibility/aspnet-core/7.0/fallback-file-endpoints.md)
-- [Output caching API changes](../core/compatibility/aspnet-core/7.0/output-caching-renames.md)
-- [Repeated XML elements include index](../core/compatibility/extensions/6.0/repeated-xml-elements.md)
+- [Breaking changes in .NET 8](../core/compatibility/8.0.md)
+- [Backslash mapping in Unix file paths](../core/compatibility/core-libraries/8.0/file-path-backslash.md)
+- [WFDEV002 obsoletion is now an error](../core/compatibility/windows-forms/8.0/domainupdownaccessibleobject.md)
+- [AnonymousPipeServerStream.Dispose behavior for HandleInheritability.Inheritable](../core/compatibility/core-libraries/8.0/anonymouspipeserverstream-dispose.md)
+- [Legacy Console.ReadKey removed](../core/compatibility/core-libraries/8.0/console-readkey-legacy.md)
+- [FileStream writes when pipe is closed](../core/compatibility/core-libraries/8.0/filestream-disposed-pipe.md)
+- [Activity operation name when null](../core/compatibility/core-libraries/8.0/activity-operation-name.md)
+- [TwoDigitYearMax default is 2049](../core/compatibility/globalization/8.0/twodigityearmax-default.md)
+- [GetFolderPath behavior on Unix](../core/compatibility/core-libraries/8.0/getfolderpath-unix.md)
+- [AesGcm authentication tag size on macOS](../core/compatibility/cryptography/8.0/aesgcm-auth-tag-size.md)
+- [RSA.EncryptValue and RSA.DecryptValue are obsolete](../core/compatibility/cryptography/8.0/rsa-encrypt-decrypt-value-obsolete.md)
+- [Anchor layout changes](../core/compatibility/windows-forms/8.0/anchor-layout.md)
+- [Top-level forms scale minimum and maximum size to DPI](../core/compatibility/windows-forms/8.0/forms-scale-size-to-dpi.md)
+- [WCF Client doesn't support .NET Standard](../core/compatibility/wcf-client/6.0/net-standard-2-support.md)
+- [Automatic RuntimeIdentifier for publish only](../core/compatibility/sdk/7.0/automatic-rid-publish-only.md)
+
+## .NET Framework
+
+### New articles
+
+- [Build apps against Microsoft.NETFramework.ReferenceAssemblies](../framework/migration-guide/reference-assemblies.md)
 
 ## .NET fundamentals
```

Comment:
Now that we've broken out the fundamentals TOC, should we make this heading more fine-grained too?

Reply:
Great suggestion.  I added #33826 so we remember.